src/components/List.jsx

```diff
@@ -20,17 +21,66 @@ export const List = ({ token }) => {
     return timeDiff < 24;
   };
 
+  const calcDaysSince = (transactionDate) => {
+    const timeNow = moment().format();
+    const date1 = moment(timeNow, 'YYYYMMDD HH:mm:ss');
+    const date2 = moment(transactionDate, 'YYYYMMDD HH:mm:ss');
+    const timeDiff = date1.diff(date2, 'days');
+
+    return timeDiff;
+  };
+
+  const cleanDate = (date) => {
+    const lastPurchasedDate = moment(date).format('MMMM Do, YYYY');
+    return lastPurchasedDate;
+  };
+
   const updateDocument = async (id) => {
     const docRef = doc(db, token, id);
+    const document = await getDoc(docRef);
+    let docData;
+    if (document) {
+      docData = document.data();
+      console.log(docData);
+    }
+    const updateEstimate = () => {
```

Comment:
A quick comment on function names: generally when I read "updateX" I assume it might be making an api request or mutating something, `getX` tells me it is going to go off and get a new value which I feel is more applicable here 

```suggestion
    const getEstimate = () => {
```